scripts/data_processing/get_data_pzo_omikron.py

```
@@ -0,0 +1,1238 @@
+        school_ids = school_ids[:limit_schools]
+
+    details: dict[str, JsonDict] = {}
+    failed_details: list[dict[str, str]] = []
+    for index, school_id in enumerate(school_ids, start=1):
+        logger.info(
+            "Pobieranie szczegółów szkoły %s (%s/%s)", school_id, index, len(school_ids)
+        )
+        try:
+            details[str(school_id)] = client.school_details(school_id)
+        except (
+            Exception
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)